### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by james-otten**
- [ ] Figure out the meshdb-cluster stuff (first time deploy)
- [ ] Finish docs
- [ ] frr + haproxy

### CODE REVIEW

File: infra/ansible/k8s_lb.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        dest: /etc/netplan/50-cloud-init.yaml
+        mode: "600"
+      become: true
+      
+    - name: Install frr
+      ansible.builtin.apt:
+        update_cache: true
+        pkg:
+          - frr
+      become: true
```
Comment (WillNilges) [inline infra/ansible/k8s_lb.yaml:30]: Combine these package installations

File: infra/ansible/k8s_lb_config.yaml
Comment (WillNilges) [inline infra/ansible/k8s_lb_config.yaml:1]: TODO: Delete file.

File: infra/ansible/k8s_lb.yaml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        enabled: true
+      become: true
+    
+    - name: Restart and enable haproxy service
+      ansible.builtin.service:
+        name: haproxy
+        state: restarted
+        enabled: true
+      become: true
+
```
Comment (WillNilges) [inline infra/ansible/k8s_lb.yaml:96]: Maybe add comment explaining the reasoning behind the steps?

File: infra/ansible/lb_config/haproxy.cfg [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	errorfile 403 /etc/haproxy/errors/403.http
+	errorfile 408 /etc/haproxy/errors/408.http
+	errorfile 500 /etc/haproxy/errors/500.http
+	errorfile 502 /etc/haproxy/errors/502.http
+	errorfile 503 /etc/haproxy/errors/503.http
+	errorfile 504 /etc/haproxy/errors/504.http
+
+frontend meshdb
+  bind {{ EXTERNAL_LISTEN_IP }}:80
+  bind {{ EXTERNAL_LISTEN_IP }}:443
```
Comment (WillNilges) [inline infra/ansible/lb_config/haproxy.cfg:38]: Might be able to delete this, especially if we use traefik. Need to test.

File: infra/ansible/lb_config/netplan_50_cloud_init.yaml.j2 [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+# network: {config: disabled}
+network:
+    version: 2
+    ethernets:
+        eth0:
+            dhcp4: no
+            dhcp6: no
+            addresses: [ "{{ inventory_hostname }}/{{ INTERNAL_NETWORK_RANGE }}" ]
+            nameservers:
+                addresses: [ "10.10.10.10", "10.10.10.11", "8.8.8.8" ]
```
Comment (WillNilges) [inline infra/ansible/lb_config/netplan_50_cloud_init.yaml.j2:14]: This might cause trouble; We'll see during the test.

File: infra/tf/mgr.tf [hunk 5]
```diff
@@ -1,9 +1,3 @@
-resource "null_resource" "ssh_key" {
```
Comment (WillNilges) [inline infra/tf/mgr.tf:1]: TODO: Come back to this at some point™

File: infra/tf/vars.tf [hunk 6]
```diff
@@ -66,7 +70,17 @@ variable "meshdb_networkrange" {
   default      = "24"
 }
 
+variable "meshdb_net_block" {
```
Comment (WillNilges) [inline infra/tf/vars.tf:73]: TODO: Investigate this further

Comment (WillNilges) [reply infra/tf/vars.tf:73]: I think this is a variable we use for FRR config.

File: infra/tf/mgr.tf (hunk 5 above)
Comment (james-otten) [reply infra/tf/mgr.tf:1]: Gone

File: infra/ansible/lb_config/haproxy.cfg (hunk 3 above)
Comment (james-otten) [reply infra/ansible/lb_config/haproxy.cfg]: certbot used for let's encrypt instead of traefik

File: infra/tf/vars.tf (hunk 6 above)
Comment (james-otten) [reply infra/tf/vars.tf:73]: Cleaned up var names

**Comment by WillNilges** [comment]
Closing the following PRs in favor of this one:
- https://github.com/nycmeshnet/meshdb/pull/368
- https://github.com/nycmeshnet/meshdb/pull/362

**Pull request closed without merging**